src/rest-server/yarn.lock

```diff
@@ -2420,6 +2420,11 @@ mocha@~5.0.0:
     mkdirp "0.5.1"
     supports-color "4.4.0"
 
+module-alias@^2.2.0:
```

Comment:
does this file need to be changed?

Reply:
yes, added a new package `module-alias` in `package.json`